samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -79,6 +79,18 @@ Recovery mode
 In the provisioned state, you can enter the recovery mode for a limited time.
 This mode allows the Android device to recover a lost provisioning key from the locator tag.
 
+DULT Motion detector active mode
```

Comment:
Pay attention to the spelling. In some places it is Motion detector and other places motion detector. Try to be consistent with the use.

Reply:
I'll stick to motion detector as that's the spelling in the official external DULT documentation.